docs/src/components/Head/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+
+interface Props {
+  title?: string;
+  hasSiteName?: boolean;
+  description?: string;
+  path: string;
+}
+
+export default function Head({ title, hasSiteName, description, path }: Props) {
+  const pageTitle = hasSiteName ? `${title} | Orbit` : title
```

Comment:
I think this is the opposite logic that we want. This is set to `isHome` for the index page, which returns true. Then it here adds ` | Orbit`. From the name `hasSiteName`, I would think it looks if the title already has the name in it. Then I would not add the name to it.

If that's true, then I'd change the logic in DocLayout. If I'm wrong, then the logic of `isHome` should change instead. 

Right now it's adding ` | Orbit` everywhere, which I think is not desired behavior.

Reply:
True, I flipped mapping of `isHome` and `hasSiteName` — `hasSiteName={!isHome}`.